crates/net/eth-wire-types/src/broadcast.rs

```diff
@@ -432,13 +432,13 @@ impl Decodable for NewPooledTransactionHashes68 {
             return Err(alloy_rlp::Error::ListLengthMismatch {
                 expected: msg.hashes.len(),
                 got: msg.types.len(),
-            })
+            });
         }
         if msg.hashes.len() != msg.sizes.len() {
             return Err(alloy_rlp::Error::ListLengthMismatch {
                 expected: msg.hashes.len(),
                 got: msg.sizes.len(),
-            })
+            });
```

Comment:
please revert unrelated formatting changes

Reply:
Got it, reverted the changes: https://github.com/paradigmxyz/reth/pull/13268/commits/2413cb2a437ba9c8af038564fb82bb0cc2588f86